.github/workflows/go.yml

```
@@ -0,0 +1,103 @@
+      with:
+        go-version: '1.22'
+        cache: true
+
+    - name: Build
+      run: go build -v ./cmd/server
+
+  ts-lint:
+    name: TypeScript Lint
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)